paymentForm.html

```diff
@@ -0,0 +1,42 @@
+<!DOCTYPE html>
+    <head>
+        <title>
+                paymentForm.html
+        </title>
+    </head>
+<body>
+    <h1>
+        <strong>Payment Form</strong>
+    </h1>
+        <p>Required fields are marked by <u>*</u></p>
+    <h3>
+        <strong>Contact Information</strong>
+    </h3>
+            <input type ='radio' value =“Mr”> Mr.<br>
```

Comment:
Keep in mind that later on not having a closing / on the input tag will cause a problem.